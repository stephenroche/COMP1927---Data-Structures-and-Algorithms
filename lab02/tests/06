995
992
986
980
955
944
937
916
897
890
879
859
849
844
844
843
840
823
818
794
761
752
751
748
747
738
737
722
700
699
694
692
669
658
648
648
644
633
615
610
607
603
538
518
502
489
483
480
473
457
456
448
413
409
396
393
393
387
381
379
376
375
339
336
327
321
310
290
268
267
253
251
236
233
229
213
202
201
177
172
170
151
146
138
137
136
132
130
127
113
91
70
55
40
39
21
19
16
14
9
