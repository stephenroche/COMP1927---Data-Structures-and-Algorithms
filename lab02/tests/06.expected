9
14
16
19
21
39
40
55
70
91
113
127
130
132
136
137
138
146
151
170
172
177
201
202
213
229
233
236
251
253
267
268
290
310
321
327
336
339
375
376
379
381
387
393
393
396
409
413
448
456
457
473
480
483
489
502
518
538
603
607
610
615
633
644
648
648
658
669
692
694
699
700
722
737
738
747
748
751
752
761
794
818
823
840
843
844
844
849
859
879
890
897
916
937
944
955
980
986
992
995
